### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 9

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#309] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322224-5cdad09f-aa8f-41a2-b252-890f255b1d9f.png)

**Comment by Exyss** [comment]
<img width="400" alt="image" src="https://user-images.githubusercontent.com/69389461/153565804-22ca68b3-984a-4524-817f-b47f3a69225e.png">

**Comment by rimaout** [comment]
### Soluzione con metodo di bernulli:

<img width="969" alt="image" src="https://github.com/user-attachments/assets/b045c9b3-f15b-4122-9512-36f0758f441f">

**Comment by ChiareDaze** [comment]
<img width="500" alt="image" src="https://github.com/user-attachments/assets/d7062ab9-db50-45ef-b6c7-e3b6681b14f3">
